.chloggen/remove_access_token_passthrough.yaml

```diff
@@ -0,0 +1,12 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+# The name of the component, or a single word describing the area of concern, (e.g. agent, clusterReceiver, gateway, operator, chart, other)
+component: gateway
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Remove `access_token_passthrough` from the gateway configuration since it has been removed from `signalfx` receiver
+# One or more tracking issues related to the change
+issues: [2122]
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:
```

Comment:
Can you add a note to add `include_metadata` in the sfx receiver.

Also, update the advanced setting with an example on how to accomplish this and link it in the changelog https://github.com/signalfx/splunk-otel-collector-chart/blob/main/docs/advanced-configuration.md#otlp-token-passthrough

Note: the current doc is about `OTLP Token Passthrough` , but we probably need to make it more generics `Token Passthrough` and have 2 examples, one fore OTLP and one for SFX receiver